src/controllers/ratingsController.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-constant-condition */
+import model from '../database/models';
+import fService from '../services/facilityService';
+
+const addRating = async (req, res, next) => {
+  let resp = '';
+  const facId = req.params.id;
+  try {
+    const facility = await fService.findFacilityById(facId);
+    if (!facility) {
+      resp = res.status(404).json({ message: res.__('Facility Not found!') });
+    } else {
+      req.body.facilityId = facId;
+      req.body.userId = req.userId;
+      const rating = await model.rating.create(req.body);
+      if (rating) resp = res.status(201).json({ message: res.__('rating created'), rating });
+    }
+  } catch (error) {
+    return res.status(500).json({ message: res.__('Bad request!') });
+  }
+  next();
+  return resp;
+};
+
+const getRatings = async (req, res, next) => {
+  let resp = '';
+  const facId = req.params.id;
+  try {
+    const facility = await fService.findFacilityById(facId);
+    const ratings = facility ? await model.rating.findAll({ where: { facilityId: facId } }) : false;
+    if (!ratings) {
+      resp = res.status(404).json({ message: res.__('Facility Not found!') });
+    } else {
+      resp = res.status(200).json({ success: true, ratings });
+    }
+  } catch (error) {
+    resp = res.status(500).json({ message: res.__('Bad request') });
+  }
+  next();
```

Comment:
Which function is next? and how do you hope to return a response after forwarding the request to the next processing function if it was there?